extension/encoding/googlecloudlogentryencodingextension/README.md

```diff
@@ -338,3 +340,36 @@ See the struct of the Cloud Audit Log payload in [AuditLog](https://cloud.google
 | `psc.psc_attachment.vpc.subnetwork_region` | `gcp.vpc.flow.private_service_connect.attachment.vpc.subnet.region` | not yet supported |
 | `psc.psc_attachment.vpc.vpc_name` | `gcp.vpc.flow.private_service_connect.attachment.vpc.name` | not yet supported |
 | `rdma_traffic_type` | `gcp.vpc.flow.remote_direct_memory_access.traffic_type` | not yet supported |
+
+
+### Cloud Armor logs
+
+[Cloud Armor logs](https://docs.cloud.google.com/armor/docs/request-logging#security_policy_log_entries) are mapped this way in the resulting OpenTelemetry log.
+
+| Original field | Log record attribute |
+|---|--- |
+| `statusDetails` | `gcp.load_balancing.status.details`|
+| `backendTargetProjectNumber` | `gcp.load_balancing.backend_target_project_number` |
+| `remoteIp` | `network.peer.address` |
+| `securityPolicyRequestData.recaptchaActionToken.score` | `gcp.armor.request_data.recaptcha_action_token.score` |
+| `securityPolicyRequestData.recaptchaSessionToken.score` | `gcp.armor.request_data.recaptcha_session_token.score` |
+| `securityPolicyRequestData.userIpInfo.source` | `http.request.header.source` |
```

Comment:
Sorry, I made a mistake previously here. This value actually says with header contains the user ip, so `http.request.header.source` wouldn't be correct here. You can revert to your previous value